samples/aspnetcore/WebAppPlugin1/WebAppPlugin1.csproj

```diff
@@ -1,11 +1,14 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
     <IsPlugin>true</IsPlugin>
+    <OutputType>Library</OutputType>
+    <MvcRazorCompileOnPublish>false</MvcRazorCompileOnPublish>
```

Comment:
Interesting. I would prefer to see if we can get MVC to use precompiled views instead of runtime compilation.